internal/domain/container/container.go

```diff
@@ -123,10 +123,10 @@ type NewContainerParams struct {
 	State                *string
 	Entrypoint           *string
 	Arguments            []string
-	Storages             storage.Storages
-	Ports                port.Ports
-	EnvironmentVariables variable.Variables
-	Secrets              secret.Secrets
+	Storages             storage.Storages   // TODO(benjaminch): use param struct instead `storage.NewStoragesParam`
```

Comment:
Comments can be removed :heavy_check_mark:

Reply:
updated comments, Params struct still needs to be used, to be introduced in a future refactoring eventually